File: Pyvvo.Logistics/ClientApp/src/app/service/Model/return.ts
import { Status } from "./status";
import { Order } from "./order";
import { User } from "./user";
import { Note } from "./Note";
import { ReturnLineItem } from "./return-line-item";

export class Return {
  id: number;
  referenceNumber: string;
  referenceNumberId: number;
  shippedOn: Date;
  receiveOn: Date;
  createdOn: Date;
  updatedOn: Date;
  startDate: Date;
  endDate: Date;
  isActive: boolean;
  sendUpdate: boolean;
  trackNumber: string;
  trackUrl: string;
  shippingCharge: string;
  status: Status;
  order: Order;
  createdBy: User;
  returnLineItems: ReturnLineItem[];
  notes: Note[];
  agent: User;
}

File: Pyvvo.Logistics.Model/Model/Return.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Pyvvo.Logistics.Model
{
    public class Return
    {
        [Required, Key] public long Id { get; set; }
        [Required] public long ShippingMethodId { get; set; }
        [Required] public long StatusId { get; set; }
        [Required] public long OrderId { get; set; }
        [Required] public long CreatedById { get; set; }
        [Required] public long AgentId { get; set; }
        public long ReferenceNumberId { get; set; }
        public string ReferenceNumber { get; set; }
        [Required] public DateTime CreatedOn { get; set; }
        public DateTime UpdatedOn { get; set; }
        public DateTime ShippedOn { get; set; }
        public DateTime ReceivedOn { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public bool IsActive { get; set; }
        public bool SendUpdate { get; set; }
        public string TrackNumber { get; set; }
        public string TrackUrl { get; set; }
        public string ShippingCharge { get; set; }
        public ShippingMethod ShippingMethod { get; set; }
        [Required] public Status Status { get; set; }
        [Required] public Order Order { get; set; }
        [Required] public User CreatedBy { get; set; }
        public User Agent { get; set; }
        public List<Note> Notes { get; set; }
        public List<ReturnLineItem> ReturnLineItems { get; set; }
        
    }
}